packages/node/src/generators/application/files/fastify/src/app/app.ts__tmpl__

```diff
@@ -1,11 +1,27 @@
+import * as path from 'path';
 import { FastifyInstance, FastifyRequest, FastifyReply } from 'fastify';
+import AutoLoad from '@fastify/autoload';
 
 /* eslint-disable-next-line */
-export interface AppOptions {
-}
+export interface AppOptions { }
 
 export async function app(fastify: FastifyInstance, opts: AppOptions) {
-  fastify.get('/', async function(request: FastifyRequest, reply: FastifyReply) {
-    return { message: 'Hello API' };
+  // Place here your custom code!
+
+  // Do not touch the following lines
+
+  // This loads all plugins defined in plugins
+  // those should be support plugins that are reused
+  // through your application
+  fastify.register(AutoLoad, {
```

Comment:
One thing Matteo Collina mentioned is that AutoLoad won't work with bundling (obviously). So not sure we should have some conditional generation here 🤔

Reply:
Or just have a separate template for bundling

Reply:
Yeah for sure it won't work with bundling. I think users need to change it themselves if they choose to do it that way, because they can always pass `--bundle` after the code is generated, which we don't have control over.